From 92162b6d1c0b53760f86c7899f4dc9f21886c8a9 Mon Sep 17 00:00:00 2001
From: Vyacheslav Bocharov <adeep@lexina.in>
Date: Thu, 1 Sep 2022 14:13:35 +0300
Subject: [PATCH 3/4] remove static: -static inline struct meson_pwm
 *to_meson_pwm(struct pwm_chip *chip) +inline struct meson_pwm
 *to_meson_pwm(struct pwm_chip *chip)

---
 drivers/pwm/pwm-meson.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pwm/pwm-meson.c b/drivers/pwm/pwm-meson.c
index 57112f438c6d..64874176e5f4 100644
--- a/drivers/pwm/pwm-meson.c
+++ b/drivers/pwm/pwm-meson.c
@@ -112,7 +112,7 @@ struct meson_pwm {
 	spinlock_t lock;
 };
 
-static inline struct meson_pwm *to_meson_pwm(struct pwm_chip *chip)
+inline struct meson_pwm *to_meson_pwm(struct pwm_chip *chip)
 {
 	return container_of(chip, struct meson_pwm, chip);
 }
-- 
2.30.2

